locationdelegation/src/main/java/com/google/androidbrowserhelper/locationdelegation/PermissionRequestActivity.java

```diff
@@ -42,7 +44,8 @@
     private static final String EXTRA_PERMISSIONS = "EXTRA_PERMISSIONS";
     private static final String EXTRA_GRANT_RESULTS = "EXTRA_GRANTED_RESULT";
 
-    private static final String LOCATION_PERMISSION = Manifest.permission.ACCESS_FINE_LOCATION;
+    private static final String[] LOCATION_PERMISSION =
```

Comment:
Can we make this into a List instead of storing it as an array and converting it into a list when we use it?

Reply:
This is also used in requesting permission, which takes an array as input. I think converting array to list is easier than convert list to array?

Reply:
Ah, fair enough.